var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -62,6 +62,13 @@ class Legion(CMakePackage):
     depends_on('papi', when='+papi')
     depends_on('zlib', when='+zlib')
 
+    # A C++ standard variant to work-around some odd behaviors with apple-clang
+    # but this might be helpful for other use cases down the road.  Legion's
+    # current development policy is C++11 or greater so we capture that aspect
+    # here.
+    cpp_stds = ["11", "14", "17", "20"]
+    variant('c++std', default="11", values=cpp_stds, multi=False)
```

Comment:
For consistency with other packages, I recommend:

```suggestion
    variant('cxxstd', default="11", values=cpp_stds, multi=False)
```